spec/requestReply_at_channel_dedicated.yaml

```diff
@@ -0,0 +1,63 @@
+channels:
```

Comment:
pro:
- address and replyAddress are cloth together 
- replyAddress and correlationId are cloth together
- simple not duplicated component.message

con:
- the address of channel.pongDynamic is to find in channel.pingDynamic but the correlation between the channels is done in operation